tools/benchmark/src/index.ts

```diff
@@ -32,6 +32,7 @@ export {
     benchmarkMemory,
     MemoryTestArguments,
     MemoryTestObjectInterface,
+    MemoryTestObjectProps,
```

Comment:
Adding this because the build fails otherwise: https://dev.azure.com/fluidframework/public/_build/results?buildId=111376&view=logs&j=3dc8fd7e-4368-5a92-293e-d53cefc8c4b3&t=bfe27d5f-9219-57c0-0d67-1b2946a32921&l=49